Sources/XCTest/XCTAssert.swift

```diff
@@ -52,25 +52,19 @@ private enum _XCTAssertionResult {
     case UnexpectedFailure(ErrorProtocol)
 
     var expected: Bool {
-        switch (self) {
-        case .UnexpectedFailure(_):
-            return false
-        default:
-            return true
+        switch self {
+        case .UnexpectedFailure(_): return false
```

Comment:
I tried to explain my intent in the commit.
When the statements inside of a case fit on one line (for all the cases in the switch), I moved them next to the case-label.
If any case required multiple lines, I added a line break.
This results in consistency within a switch-statement, but not necessarily across different switches.

I use this style in my own projects, as it's consistent, but also reduces the size of switches, with short expressions in each case.